### PR TITLE
Configurable CNI support

### PR DESCRIPTION
**Pull request by aaronjwood**

### CODE REVIEW

File: cassandra-executor/apache-cassandra-2.2.5/conf/cassandra-rackdc.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #DCOS got yo Cassandra!
-#Thu Oct 13 15:47:50 EDT 2016
```
Comment (thanset) [inline cassandra-executor/apache-cassandra-2.2.5/conf/cassandra-rackdc.properties]: I would remove this file from the commit, not needed.

**Comment by thanset** [comment]
This looks good.  Do we want to squash commits on feature branches? I think that would keep development branch's log cleaner that way. @aaronjwood

**Comment by aaronjwood** [comment]
Sure, will squash this up.

**Comment by aaronjwood** [comment]
I'm gonna reset this from the `development` branch since I needed to make more changes for upstream https://github.com/mesosphere/dcos-cassandra-service/pull/287

Might be better to just pull this in from upstream once it's merged there.

**Pull request merged**